repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/config/RepairnatorConfig.java

```diff
@@ -74,6 +74,9 @@
     private Duration summaryFrequency;
     private String[] notifySummary;
     private int numberOfPatchedBuilds;
+    private boolean kubernetesMode;
```

Comment:
Actually, instead of a boolean, we need an enumeration PIPELINE_MODE { DOCKER, KUBERNETES, NOPE }

Reply:
Hi @monperrus , it is fixed :) .